src/main/java/kr/codesqaud/cafe/board/controller/CommentController.java

```diff
@@ -16,15 +18,18 @@ public CommentController(CommentService commentService) {
     }
 
     @PostMapping("/board/{postId}/comment")
-    public String writeComment(@PathVariable Long postId, @ModelAttribute CommentWriteForm commentWriteForm, @SessionAttribute("sessionUser") SessionUser sessionUser) {
-        commentService.write(commentWriteForm, sessionUser.getUserName());
-        return "redirect:/board/" + postId;
+    public CommentResponse writeComment(@PathVariable Long postId, @ModelAttribute CommentWriteForm commentWriteForm, @SessionAttribute("sessionUser") SessionUser sessionUser) {
```

Comment:
SessionAttribute value 네이밍이 같으면 생략가능합니다.
```suggestion
    public CommentResponse writeComment(@PathVariable Long postId, @ModelAttribute CommentWriteForm commentWriteForm, @SessionAttribute SessionUser sessionUser) {
```